Bottom: a68a4d08c4aa027858cc023f476306031723446b
Top:    a0ffc0711c6f939cf434e38fb906c2ebe3975d40
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-13 15:49:16 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/man/automount.8 b/man/automount.8
index 48f5c5a..25282f9 100644
--- a/man/automount.8
+++ b/man/automount.8
@@ -57,7 +57,7 @@ Run the daemon in the foreground and log to stderr instead of syslog."
 Enables the use of ramdom selection when choosing a host from a
 list of replicated servers.
 .TP
-.I "\-m, \-\-dumpmaps [<map type> <map name]"
+.I "\-m, \-\-dumpmaps [<map type> <map name>]"
 With no parameters, list information about the configured automounter
 maps, then exit.
